app/src/main/java/com/myperishableplanner/v21001/ItemViewModel.kt

```diff
@@ -12,26 +12,20 @@ import com.google.firebase.firestore.FirebaseFirestoreSettings
 import com.google.firebase.firestore.FirebaseFirestore
 import com.myperishableplanner.v21001.dto.ItemDetail
 
-
 class ItemViewModel (var itemService: IItemService = ItemService()): ViewModel() {
 
+    private var firestore : FirebaseFirestore = FirebaseFirestore.getInstance()
+    var items : MutableLiveData<List<Item>> = MutableLiveData<List<Item>>()
 
-        var items : MutableLiveData<List<Item>> = MutableLiveData<List<Item>>()
-
-        private lateinit var firestore : FirebaseFirestore
```

Comment:
Assign variable FirebaseFirestore.getInstance() right away instead of in the init.